### PR TITLE
redirect from login page

### PR DESCRIPTION
**Pull request by raniarusly**
add logic to redirect to recommendation map from login page when user is logged in

### CODE REVIEW

File: src/main/webapp/scripts/common.js [hunk 1]
```diff
@@ -24,17 +24,23 @@ window.onscroll = function() {
...
-      const loginPageURL = window.location.protocol + "//" + window.location.host;
       if(window.location.href !== loginPageURL) {
         window.location.href = loginPageURL;
       }
 
       // On the login page put the link to login with email on the login button 
       document.getElementById("login-link").href = details.loginURL;
     }
+    else if (window.location.href === loginPageURL) {
+        console.log(window.location.href);
```
Comment (lornaarmstrong) [inline src/main/webapp/scripts/common.js]:
```suggestion
      console.log(window.location.href);
```

**Review by lornaarmstrong — APPROVED** [review]
LGTM!

File: src/main/webapp/scripts/common.js [hunk 2]
```diff
@@ -24,17 +24,23 @@ window.onscroll = function() {
 
 window.onload = function() {
   fetch("/login").then(result => result.json()).then((details) => {
+    const loginPageURL = window.location.protocol + "//" + window.location.host + "/";
+    console.log(window.location.href);
```
Comment (MostafaGabriel) [inline src/main/webapp/scripts/common.js]: This was only for debugging purposes I assume, shall we remove it? Same below :)

File: src/main/webapp/scripts/common.js [hunk 3]
```diff
@@ -24,17 +24,23 @@ window.onscroll = function() {
...
     if (!details.isUserLoggedIn) {
-        
       // If user is not logged in then redirect to the login page, if not already there
```
Comment (MostafaGabriel) [inline src/main/webapp/scripts/common.js]: Please move this comment above the outer `if` as it's describing the whole block

**Review by MostafaGabriel — APPROVED** [review]
Thanks Rania :)

**Pull request merged**